docs/reference/indices/forcemerge.asciidoc

```diff
@@ -10,6 +10,12 @@ This call will block until the merge is complete. If the http connection is
 lost, the request will continue in the background, and any new requests will
 block until the previous force merge is complete.
 
+[WARNING]
+Force merge should only be called against *read-only indices*. Running 
```

Comment:
Just do `WARNING: Force merge...` and our documentation processor will make a nice warning block out of this paragraph.